src/element/list.rs

```diff
@@ -19,21 +19,48 @@ use std::fmt::{Display, Formatter};
 /// # }
 /// ```
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct List {
-    pub(super) children: Vec<Element>,
+pub struct List(pub Sequence);
+
+impl List {
+    pub(crate) fn new(elements: impl Into<Sequence>) -> Self {
+        List(elements.into())
+    }
 }
 
+// impl Deref for List {
+//     type Target = Sequence;
+//
+//     fn deref(&self) -> &Self::Target {
+//         &self.0
+//     }
+// }
+
 impl List {
-    pub(crate) fn new(children: Vec<Element>) -> Self {
-        Self { children }
+    pub fn builder() -> SequenceBuilder {
+        Sequence::builder()
+    }
```

Comment:
It is a little unfortunate that you can write this code now:
```rust
List::builder().build_sexp();
```
Would it be worth putting a type parameter into `SequenceBuilder` to prevent this from happening? I'm putting a comment over by SequenceBuilder with an example of what I mean.

Reply:
Oof, yeah, that's unfortunate. Good catch.